CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Traffic Ops: Added validation to ensure that the cachegroups of a delivery services' assigned ORG servers are present in the topology
 
 ### Fixed
+- [#5382](https://github.com/apache/trafficcontrol/issues/5382) - Fixed helptext for "Max DNS Answers" field with respect to DNS, HTTP, Steering Delivery Service
```

Comment:
how about - "Fixed API documentation and TP tooltip text for..."

Reply:
I changed the helptext and not tooltip. Have edited changelog accordingly.